front/pages/api/v1/w/[wId]/data_sources/[name]/documents/[documentId]/index.ts

```diff
@@ -276,6 +276,7 @@ async function handler(
         sourceUrl,
         text: req.body.text,
         credentials,
+        suppressDocumentOutput: req.body.suppress_document_output === true,
```

Comment:
NIT: shall we add it somewhere in here?  https://docs.dust.tt/documents#create-a-document

Reply:
added: [documentation](https://github.com/dust-tt/dust/pull/897/commits/93b933bd5c42dcff276e5b0f25ec7432141e7fd3)